packages/react/src/components/VideoPlayer/VideoPlayer.js

```diff
@@ -17,10 +18,13 @@ const { prefix } = settings;
 /**
  * VideoPlayer component
  *
- * @param {string} videoId Kaltura video id
+ * @param {object} props props object
+ * @param {string} props.type theming options
```

Comment:
General question, is `type` the right prop name for this or we want to rename to something else?

cc: @andysherman2121 @kennylam @reesedownes @annawen1 @RaphaelAmadeu @guilhermelMoraes

Reply:
should we just have it as a boolean called `inverse`?

Reply:
Updated to `inverse`